packages/browser-wallet/src/popup/page-layouts/MainLayout/Header/Header.tsx

```diff
@@ -129,7 +129,7 @@ export default function Header({ onToggle }: Props) {
                     {isHomePage || (
```

Comment:
According to the design, it seems that the different account pages should have exit buttons within themselves, but that this should not appear?

Reply:
Yes, you're right!